app/core/ts/tools/SelectionTool.ts

```diff
@@ -209,6 +213,21 @@ export class SelectionTool extends DefaultTool {
             return true;
         }
 
+        // Fit to screen command
+        if (key === F_KEY) {
+            const initialPos = this.camera.getPos();
+            const initialZoom = this.camera.getZoom();
+
+            // Fit to selections, if any; otherwise, fit all CullableObjects
+            const objs = this.selections.size === 0
+                ? (this.designer.getObjects() as CullableObject[]).concat(this.designer.getWires())
+                : this.getSelections().filter(o => o instanceof CullableObject) as CullableObject[];
+
+            FitCamera(this.camera, objs, FIT_PADDING_RATIO);
+            this.action.add(MoveCameraAction.postMoveCameraAction(this.camera, initialPos, initialZoom));
```

Comment:
Instead of having a `postMoveCameraAction`, we have a few options:
1. `FitCamera` returns a `MoveCameraAction` to the desired position and zoom
2. `FitCamera` returns a tuple of the final position and zoom, {pos, zoom} and then we create the regular action

I'll leave you to decide this, but I personally like the latter.

Reply:
I see how 1 would work, but `FitCamera` already kind of returns the final position and zoom in that it modifies the original camera object to have the final position and zoom.

Another thing I could do is make a copy of the camera before `FitCamera` and then used the regular `MoveCameraAction` constructor using that copy and the final position and zoom. (This might be best done by adding a `Camera.copy` method.)

Reply:
I don't think copying the entire camera is necessary.

I like 2) because it turns FitCamera from a function that changes the camera to one that just calculates some numbers which is usually better since it's easier to test. And if you go with 2) we should rename the function GetBestFit or something

Reply:
Ok, I'll do 2